schema/base/document-transition.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://schema.dash.org/dpp-0-4-0/base/document-transition",
+  "type": "object",
+  "properties": {
+    "$id": {
+      "type": "string",
+      "minLength": 42,
+      "maxLength": 44,
+      "pattern": "^[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]+$"
+    },
+    "$type": {
+      "type": "string"
+    },
+    "$action": {
+      "type": "number",
+      "enum": [1, 2, 4]
```

Comment:
Let’s start with 0?

Reply:
@shumkov Maybe move it to another story?

Reply:
Yeah, we can, but I think we just need to update schema here and constants. Let's try to do that when all other stuff is ready.